src/views/cic/confirm-details.html

```diff
@@ -48,7 +48,7 @@ <h1 class="govuk-heading-l">{{ translate("checkDetails.title") }}</h1>
               items: [
                 {
                   href: "/enter-date-birth/edit",
-                  text: "Change",
+									text: translate("checkDetails.changeLink"),
                   visuallyHiddenText: "date of birth"
```

Comment:
This is for screen readers it should probably change too right?

Reply:
Done